src/transforms/dimensionalize.jl

```diff
@@ -130,55 +130,63 @@ combinedim(ctx, a::NoDimension, b) = b
     stop
 end
 
+@kwdef struct ContinuousExtent
+    start
+    stop
+end
+
+is_continuous_extent(x) = false # generic
```

Comment:
I feel like any place where we use this should probably be converted to a named function that specializes on the type of the extent

Reply:
e.g. `measure` or `get_eps`